req.txt

```diff
@@ -1,4 +1,5 @@
-telethon>=1.37.0
-requests>=2.28.1
-urllib3>=1.26.12
-python-dotenv
+telethon>=1.37.0
+requests>=2.28.1
+urllib3>=1.26.12
+python-dotenv
+pytz

```

Comment:
why did you rename `requirements` to `req` and what's the point of the `pytz` lib?